AdyenComponents/BACS Direct Debit/Scenes/Input/BACSDirectDebitInputFormView.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+internal protocol BACSDirectDebitInputFormViewProtocol {
+    func add<T: FormItem>(item: T?)
+    func displayValidation()
+}
+
+internal class BACSDirectDebitInputFormView: FormViewController, BACSDirectDebitInputFormViewProtocol {
```

Comment:
Naming a controller as view can be confusing later on. We treat them as the same time to time but yeah :)

Reply:
Renamed to `BACSDirectDebitInputFormViewController`